ush/config_defaults.sh

```diff
@@ -2107,12 +2107,16 @@ DO_SMOKE_DUST="FALSE"
 # DO_PARALLEL_PRDGEN:
 # Flag turn on parallel wgrib2 runs in prdgen .
 #
+# DO_SAVE_INPUT:
+# Decide whether or not to save input along with saving restart files  
+#
 DO_RETRO="FALSE"
 DO_SPINUP="FALSE"
 LBCS_ICS_ONLY="FALSE"
 DO_POST_SPINUP="FALSE"
 DO_POST_PROD="TRUE"
 DO_PARALLEL_PRDGEN="FALSE"
+DO_SAVE_INPUT="FALSE"
```

Comment:
It looks like we can safely remove this "DO_SAVE_INPUT" variable which is not used at all.